screen-orientation/browsing-contexts.html

```diff
@@ -63,18 +62,25 @@
       "popup test",
       "width=500,height=300"
     );
-    await test_driver.bless("request full screen", () => {
-      return document.documentElement.requestFullscreen();
+    const loadPromise = new Promise(r => {
+      if (popWindow.document.readyState === "complete") {
+        r();
+      }
+      popWindow.onload = () => r();
     });
+    await loadPromise;
     popWindow.onload = function navigation() {
```

Comment:
This is now redundant, I guess. Eventually, after orientation change, you want to navigate to: 
```
popWindow.location.href = "https://not-web-platform.test:8443";
```

Which should drop popup out of fullscreen, and change presumedly the orientation.